dandiapi/zarr/tasks/__init__.py

```diff
@@ -131,7 +132,8 @@ def ingest_zarr_archive(
 
         # Instantiate updater and add files as they come in
         empty = True
-        updater = SessionZarrChecksumUpdater(zarr_archive=zarr)
+        queue = ZarrChecksumModificationQueue()
+        print(f'Fetching files for zarr {zarr.zarr_id}...')
```

Comment:
If we want to log this, I think `logger.info` is more appropriate than a print statement
```suggestion
        logger.info(f'Fetching files for zarr {zarr.zarr_id}...')
```